ravenframework/Optimizers/parentSelectors/parentSelectors.py

```diff
@@ -42,7 +47,7 @@ def rouletteWheel(population,**kwargs):
   """
   # Arguments
   pop = population
-  fitness = kwargs['fitness']
+  fitness = np.array([item for sublist in datasetToDataArray(kwargs['fitness'], list(kwargs['fitness'].keys())).data for item in sublist])
```

Comment:
Please specify dtype following: https://numpy.org/neps/nep-0034-infer-dtype-is-object.html